index.js

```diff
@@ -0,0 +1,207 @@
+const select = document.querySelector("select");
+const section = document.querySelector("#display-info");
+const form = document.querySelector("form");
+
+const getFilms = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+    debugger;
+    res.data.forEach((film) => {
+      const option = document.createElement("option");
+      option.innerText = film.title;
+      select.appendChild(option);
+      debugger;
+    });
+  } catch (error) {
+    console.log(error);
+  }
+};
+getFilms();
+
+// const createSection = async () => {
+//   try {
+//     const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+//     res.data.forEach((film) => {
+//   debugger
+//       const title = document.createElement("h3");
+//       const releaseYear = document.createElement("p");
+//       const description = document.createElement("p");
+//       title.innerText = film.title;
+//       releaseYear.innerText = film.release_date;
+//       description.innerText = film.description;
+//       section.appendChild(title);
+//       section.appendChild(releaseYear);
+//       section.appendChild(description);
+//     //   section.innerText = "";
+//     });
+//   } catch (error) {
+//     console.log(error);
+//   }
+// };
+
+// createSection();
+
+select.addEventListener("click", async (e) => {
+    debugger
+    try {
+        const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+        res.data.forEach((film) => {
+      debugger
+          const title = document.createElement("h3");
+          const releaseYear = document.createElement("p");
+          const description = document.createElement("p");
+          title.innerText = film.title;
+          releaseYear.innerText = film.release_date;
+          description.innerText = film.description;
+          section.appendChild(title);
+          section.appendChild(releaseYear);
+          section.appendChild(description);
+        //   section.replace(title);
+        //   section.innerText = "";
+        });
+      } catch (error) {
+        console.log(error);
+      }
+});
```

Comment:
The event listener here should be `change`. 
The api call here should be to a [single film using its id](https://ghibliapi.herokuapp.com/#tag/Films%2Fpaths%2F~1films~1%7Bid%7D%2Fget). To get the id of the film that was chosen by the select dropdown we can utilize the `event` object ==> `e.target.value`. The `value` property was used before to store the id of that movie. Once we get the data we just reassign the `innerText` for the title, release date, and description.